### ISSUE TITLE
Error: Cannot find module '@serverless/utils/log' on 3.1.0 release with serverless 3.13.0

### ISSUE THREAD

**Issue by coreyk** [newrelic/serverless-newrelic-lambda-layers#232]
Updated `serverless-newrelic-lambda-layers` from `3.0.0` to `3.1.0` and now I can't deploy:

```
Environment: linux, node 17.9.0, framework 3.13.0, plugin 6.2.1, SDK 4.3.2
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Error: Cannot find module '@serverless/utils/log'
Require stack:
- /tmp/build/99393d03/main-branch/node_modules/serverless-newrelic-lambda-layers/dist/index.js
- /usr/local/share/.config/yarn/global/node_modules/serverless/lib/utils/import-module.js
- /usr/local/share/.config/yarn/global/node_modules/serverless/lib/classes/plugin-manager.js
- /usr/local/share/.config/yarn/global/node_modules/serverless/lib/serverless.js
- /usr/local/share/.config/yarn/global/node_modules/serverless/scripts/serverless.js
- /usr/local/share/.config/yarn/global/node_modules/serverless/bin/serverless.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/tmp/build/99393d03/main-branch/node_modules/serverless-newrelic-lambda-layers/dist/index.js:19:20)
    at Module._compile (node:internal/modules/cjs/loader:1099:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at module.exports (/usr/local/share/.config/yarn/global/node_modules/serverless/lib/utils/import-module.js:5:12)
    at PluginManager.requireServicePlugin (/usr/local/share/.config/yarn/global/node_modules/serverless/lib/classes/plugin-manager.js:171:14)
    at PluginManager.resolveServicePlugins (/usr/local/share/.config/yarn/global/node_modules/serverless/lib/classes/plugin-manager.js:198:29)
    at async PluginManager.loadAllPlugins (/usr/local/share/.config/yarn/global/node_modules/serverless/lib/classes/plugin-manager.js:136:36)
    at async Serverless.init (/usr/local/share/.config/yarn/global/node_modules/serverless/lib/serverless.js:141:5)
    at async /usr/local/share/.config/yarn/global/node_modules/serverless/scripts/serverless.js:580:7
```

**Comment by mrickard** [comment]
@coreyk This should be fixed with the just-released v3.1.1. Please try it and let us know if that fixes it!

**Comment by mrickard** [comment]
Closing; resolved with release v3.1.1
